### DISCUSSION TITLE
Reorder dropdown list of fields in query builder

### DISCUSSION THREAD

**Discussion by drew-wallace** [specify/specify7#5506] [Feature Requests]
A user requested the ability to reorder fields within the dropdown menu after selecting a table in the query builder.

Click Query>CO>"+">dropdown menu

**Comment by maxpatiiuk** [comment]
The fields are ordered by their label.

Thus, as a workaround, the field order can be changed by modifying the field labels or including a number in front of the field label.

**Comment by maxpatiiuk** [comment]
If anyone is to fix this, make sure to not limit this just to query builder, but to fix it in the place where fields get ordered by the label (look at data model fetching code)

**Comment by grantfitzsimmons** [comment]
<img width="277" alt="image" src="https://github.com/specify/specify7/assets/37256050/633821a7-65f9-4e4c-9a5d-0fcc3ab621b0">

This is a request looking for the ability to change the order of fields in the dropdown in a query item
